genesyscloud/tfexporter/genesyscloud_resource_exporter.go

```diff
@@ -327,6 +327,7 @@ func (g *GenesysCloudResourceExporter) retrieveGenesysCloudObjectInstances() dia
 		go func(resType string, exporter *resourceExporter.ResourceExporter) {
 			defer wg.Done()
 
+			log.Printf(`Getting exported resources for [%s}]`, resType)
```

Comment:
Should this be [%s] ? Looks like the } might be accidental

Reply:
@bbbco Can you quick fix this and then we can get this merged

Reply:
thx! fixed!